max_outliers: 20